api/tests/acceptance/application/student-user-association-controller_test.js

```diff
@@ -52,7 +52,33 @@ describe('Acceptance | Controller | Student-user-associations', () => {
       expect(response.statusCode).to.equal(204);
     });
 
-    it('should return an 404 NotFoundError error if we don’t find a student to associate', async () => {
+    it('should return an 404 NotFoundError error if we don’t find a student to associate because birthdate does not match', async () => {
```

Comment:
Pour éviter de surcharger le it et donner le bon contexte:

`context('when no student birthdate matches', async () => {
  // context
  it('should return a NotFoundError', async () => {
    // expect
  }
})`